docs/introduction_tutorial/intro_tutorial_part3.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>The first <code>recipe</code> refers to the name of the module (<code>recipe.py</code> ). The second <code>recipe</code> refers to the name of the Recipe object inside that module. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>